app/src/main/res/layout/content_settings_autofill.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) 2018 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Nit: This section is called context_settings_autofill atm. Might benefit with a better naming once sync and back up is implemented since it is not going to be just autofill anymore https://www.figma.com/proto/PQyk79SourySnuA2qMkFTa/Settings-Page?page-id=107%3A8445&node-id=108%3A10037&viewport=241%2C48%2C0.25&scaling=min-zoom